workspaces/linux-host/Makefile

```diff
@@ -9,13 +9,14 @@
 # See the `LICENSING.markdown` file in the Veracruz root directory for
 # licensing and copyright information.
 
-.PHONY: all build clean clean-cargo-lock default doc fmt \
+.PHONY: all build install clean clean-cargo-lock default doc fmt \
 	test-dependencies test-collateral test-server test-client veracruz-test
 
 default: build
 
 WORKSPACE_DIR = $(abspath ..)
 OUT_DIR ?= $(abspath test-collateral)
+BIN_DIR ?= /usr/local/cargo/bin
 MEASUREMENT_FILE = $(abspath ../linux-runtime/css-linux.bin)
 MEASUREMENT_PARAMETER = --css-file $(MEASUREMENT_FILE)
 ENCLAVE_BINARY_PATH = $(abspath $(WORKSPACE_DIR)/linux-runtime/runtime_manager_enclave)
```

Comment:
I think the `RUNTIME_ENCLAVE_BINARY_PATH` thing should perhaps be added to `TEST_PARAMETERS`.